src/store/modules/nft.js

```diff
@@ -131,16 +157,31 @@ const getters = {
     return sorted;
   },
 
-  getCollectedNFTSorter: (_, getters) => ({
+  getCollectedNFTSorter: (state, getters) => ({
     nfts,
     nftOwner,
     orderBy,
     order = ORDER.DESC,
+    enableFeaturedAndHidden,
   }) => {
-    const sorted = [...nfts].sort((nA, nB) => {
+    const filtered = enableFeaturedAndHidden
+      ? nfts.filter(
+          nft => !state.userNFTClassHiddenSetMap[nftOwner]?.has(nft.classId)
+        )
+      : [...nfts];
```

Comment:
I think can refactor the filter logic part into getters
```suggestion
      ? getters.filterNFTClassListWithState(nfts)
      : []
```

Reply:
Should it be a function like this?
```
filterNFTClassListWithState: state => nfts =>
    nfts.filter(
      nft => !state.userNFTClassHiddenSetMap[collector]?.has(nft.classId)
    ),
```

Reply:
Yes seem so